src/Containers/FileUpload/IsFailureIsSuccessPanel.tsx

```diff
@@ -1,16 +1,16 @@
 import React, { useState, useEffect } from 'react';
 import { TextLayout } from '@Layouts';
 import { StyledIcon } from '@styled-icons/styled-icon';
-import { Container, Icon } from './StyledComponents';
+import { Container, Icon, IContainerProps } from './StyledComponents';
```

Comment:
Does this component have its own story? Where is it?

Reply:
no, I don't have a story for this styled component

Reply:
Add a story for this component